libs/application/templates/marriage-conditions/src/fields/Done/SpouseDone.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { Box, BulletList, Bullet } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { CustomField, FieldBaseProps } from '@island.is/application/types'
+import { Text } from '@island.is/island-ui/core'
+import { m } from '../../lib/messages'
+import { formatText } from '@island.is/application/core'
+
+interface PropTypes extends FieldBaseProps {
+  field: CustomField
+}
+
+export const SpouseDone = ({ application }: PropTypes): JSX.Element => {
+  const { formatMessage } = useLocale()
+  const bullets = [m.spouseBullet1, m.spouseBullet2, m.spouseBullet3]
+
+  return (
```

Comment:
For the sake of avoiding custom fields, could this have been solved with a description field with markdown?